tests/lib/rules/no-children-prop.js

```diff
@@ -15,6 +15,7 @@ var RuleTester = require('eslint').RuleTester;
 var parserOptions = {
   ecmaVersion: 6,
   ecmaFeatures: {
+    experimentalObjectRestSpread: true,
```

Comment:
Yes, this PR adds it here.  But there are eight other tests that include `experimentalObjectRestSpread` in their default `parserOptions` just like this one does.

If it's important to only set that option for certain test cases, then I think we should make that consistent across all of the tests that use that option.  That's why I suggested a separate PR to make that change.